### DISCUSSION TITLE
K8S 1.29 support

### DISCUSSION THREAD

**Discussion by Lumin343** [kubernetes-sigs/cluster-api#9955] [Q&A] [answered]
I was wondering when CAPI will be able to officially support k8s 1.29? Currently it's listed as only supporting up to 1.28.x. Is it possible to use the latest version of CAPI to spin up a k8s 1.29 cluster on GCE? I am currently unable to get it going.

**Comment by sbueringer** [comment, marked as answer]
Here is the corresponding tracking issue: https://github.com/kubernetes-sigs/cluster-api/issues/9578

1.29 support is already implemented on the main and release-1.6 branches and 1.29 will be officially supported with Cluster API v1.6.1 (tentative release date: 16th January 2024)

I don't know if there is anything specifically that has to be done in CAPG.

**Comment by fabriziopandini** [comment]
1.29 is supported snice ~1yr now

**Comment by fabriziopandini** [comment]
closing
